modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/price/test/CommerceProductPriceCalculationV2Test.java

```diff
@@ -1936,8 +1950,11 @@ public void testCalculatePriceWithParentPriceEntry() throws Exception {
 		CommerceMoney finalPriceCommerceMoney =
 			commerceProductPrice.getFinalPrice();
 
+		BigDecimal finalPriceCommerceMoneyPrice =
+			finalPriceCommerceMoney.getPrice();
+
 		Assert.assertEquals(
-			cpInstancePrice, finalPriceCommerceMoney.getPrice());
+			cpInstancePrice, finalPriceCommerceMoneyPrice.stripTrailingZeros());
```

Comment:
in CommerceDiscountV2Test.java we are stripping the zeros also on the first parameter
```
Assert.assertEquals(
  price.stripTrailingZeros(),
  finalPriceCommerceMoneyPrice.stripTrailingZeros());
```
is it correct that here no?